app/assets/stylesheets/keyboard_tools.scss

```diff
@@ -9,6 +9,9 @@
     position: fixed;
     left: 1rem;
     bottom: 1rem;
+    z-index: 2305843009213693951; /* This is the largest Mersenne-prime shown in full on Wikipedia and should be

```

Comment:
nitpick (non-blocking): magic numbers are bad - explanation helps, but better overall to choose something that's clearly "arbitrary high number", like `9999999999`, rather than something oddly-specific that _could_ mean something if the explanation wasn't there

Reply:
Will keep in mind next time, but - as you said this is non-blocking - I'll keep this beautiful prime number here as a little code easter-egg.